plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java

```diff
@@ -34,7 +34,7 @@ private MLCommonsSettings() {}
     public static final Setting<Long> ML_COMMONS_MONITORING_REQUEST_COUNT = Setting
         .longSetting(
             "plugins.ml_commons.monitoring_request_count",
-            10000,
+            100,
```

Comment:
Is this reduction from 10000 to 100 due to the tuning after performance tests result?

Reply:
This part is mainly for testing/monitoring purpose, big value will use more memory. So tune the default value to a smaller number. User can tune it if they want to monitor more requests.